test_build.py

```diff
@@ -70,3 +70,7 @@
 
 from skimage.io import imread
 print("skimage ok")
+
+from google.cloud import bigquery
+client = bigquery.Client()
+print("bigquery ok")
```

Comment:
Should we 1) set `KAGGLE_PROXY_DATA` here, so that the `sitecustomize` logic actually gets interpreted and 2) actually verify that the bigquery patch works as expected (it tries to connect to `KAGGLE_PROXY_URL` and passes on `X-KAGGLE-PROXY-DATA`)?

Reply:
Ah, great point.